tabs/tab_logit.py

```
@@ -629,30 +662,50 @@ def ui_forest_tabs():
+                style="color: #999; text-align: center; padding: 20px;"
+            )
+        html_str = plotly_figure_to_html(
+            res['fig_crude'],
+            div_id="plot_logit_forest_crude",
+            include_plotlyjs='cdn',
+            responsive=True
+        )
+        return ui.HTML(html_str)
 
     @render.download(filename="logit_report.html")
     def btn_dl_report():
```

Ruff (0.14.10)

Missing return type annotation for private function `btn_dl_report`

(ANN202)